Resources/config/events.xml

```diff
@@ -54,5 +55,11 @@
         <service id="fos_comment.listener.closed_threads" class="%fos_comment.listener.closed_threads.class%">
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="fos_comment.listener.dynamic_relations" class="%fos_comment.listener.dynamic_relations.class%">
+            <argument /> <!-- %fos_comment.model.thread.class% (value is dependent on whether using ORM or ODM) -->
+            <argument type="service" id="annotation_reader" />
+            <tag name="doctrine.event_subscriber" />
```

Comment:
this should be registered only when using the ORM as storage backend. Otherwise it could create weird issues in case someone uses the ODM as storage but still use DBAL (and maybe the ORM) in the same project